### DISCUSSION TITLE
Dokku plugins from a monorepo

### DISCUSSION THREAD

**Discussion by IgnisDa** [dokku/dokku#5000] [Ideas]
I am working on a project that will need a few dokku plugins to work. This project is itself a monorepo. However `dokku plugin:install ...` expects the directory to be a single plugin, and this is preventing me from making a monorepo with just dokku plugins. I am aware that I can write custom bash scripts to fix this but I think this should be managed by dokku itself.

What do you all think?

I can work on a PR implementing this feature.

**Comment by josegonzalez** [comment]
Sorry what exactly are you trying to do? Deploy an app, or install multiple plugins from a directory?

**Comment by IgnisDa** [comment]
I want to have multiple plugins in a single repository. It is not possible right now with the existing `plugin:install` command.
